src/osdatahub/requests_wrapper.py

```diff
@@ -0,0 +1,32 @@
+import requests
+
+
+def check_length(func):
+    def wrapper(*args, **kwargs):
```

Comment:
I'd strongly recommend including a docstring for this function, maybe even linking to the blogpost. This could confuse folks who don't know about the bug

Reply:
Good shout - docstrings added